File: ArrayAndList/check_array_sorted/check_array.py
arr=[1,2,3,4,5,7,6]
n=len(arr)

is_sorted=True
for i in range(n-1):
    if(arr[i]>arr[i+1]):
        is_sorted=False
        break
if is_sorted:
    print("sorted")
else:
    print("unsorted")

File: ArrayAndList/check_array_sorted/check_array.java
package ArrayAndList.check_array_sorted;

public class check_array {
    public static void main(String[] args) {
        int[] arr={1,2,3,4,5,8,7};
        int n=arr.length;
        boolean is_sorted=true;
        for(int i=0;i<n-1;i++){
            if(arr[i]>arr[i+1]){
                is_sorted=false;
                break;

            }
            

        }
        if(is_sorted) System.out.print("sorted");
        else System.out.print("unsorted");
    }
}
